src/websockets/frames.py

```diff
@@ -52,48 +52,69 @@ class Opcode(enum.IntEnum):
 CTRL_OPCODES = OP_CLOSE, OP_PING, OP_PONG
 
 
-# See https://www.iana.org/assignments/websocket/websocket.xhtml
-CLOSE_CODES = {
-    1000: "OK",
-    1001: "going away",
-    1002: "protocol error",
-    1003: "unsupported type",
+class CloseCode(enum.IntEnum):
```

Comment:
I would suggest decorating this with [`@enum.unique`](https://docs.python.org/3/library/enum.html#enum.unique). This will prevent accidental aliasing or typos where the same number is used on two enum members.

Reply:
No point burning CPU to check this every time someone imports websockets. The list is never going to change anyway.